CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   that would become empty after a rebase.
 
 ### Fixed bugs
+  
+* `jj init --git` in an existing Git repository now errors and exits rather than creating a second Git store.
```

Comment:
nit: move to "Breaking changes"?

Reply:
I was debating exactly that when I added it and was curious what folks that! It could see it going either way: on the one hand, anyone who *is* relying on the ability to `jj init --git .` in a repo with an existing Git store will be broken by this; on the other hand, (a) it is extremely unlike anyone is doing that, and (b) the behavior in that case is basically broken, so they should almost certainly be doing something else. (Whether any given bug fix is a breaking change is perhaps the most arguable part of SemVer. 😂)

I'm happy to defer to you all on which spot it should go in!

Reply:
I think the original behavior is correct (though it was a footgun), so it's not a bug. OTOH, I agree with you that this change isn't so "breaking". So I'm okay with either way.

Reply:
Moved to “Breaking changes” – it’s clear enough that way, and I am also fine categorizing it as not-a-bug.